client/src/components/v2/CameraRecordingStatus.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useCameraRecordingStatus } from 'api/v2/camera';
+import styles from './CameraRecordingStatus.module.css';
+
+const defaultStatus = 'Waiting for status...';
+
+/**
+ * @typedef {object} CameraRecordingStatusProps
+ * @property {'primary'|'secondary'} device Camera screen
+ */
+
+/**
+ *  Status of a camera
+ *
+ * @param {CameraRecordingStatusProps} props Props
+ * @returns {React.Component<CameraRecordingStatusProps>} Component
+ */
+export default function CameraRecordingStatus({ device }) {
+  const payload = useCameraRecordingStatus(device);
+
+  return (
+    <div>
+      {
+        payload
+          ? Object.keys(payload)
+            .map((field) => (
+              <div
+                className={`${styles.bottomMargin} ${styles.flex}`}
```

Comment:
The `classnames` module is installed and used just for this purpose (to combine multiple classes).

Reply:
Combined `bottomMargin` and `flex` styles to `statusRow`
869d24f